adafruit_ssd1306.py

```diff
@@ -130,6 +130,12 @@ def text(self, string, xpos, ypos, col=1):
         """Place text on display"""
         self.framebuf.text(string, xpos, ypos, col)
 
+    #pylint: disable-msg=too-many-arguments
+    def line(self, xpos0, ypos0, xpos1, ypos1, col=1):
+        """Draw a line from initial to final point"""
+        self.framebuf.line(xpos0, ypos0, xpos1, ypos1, col)
+    #pylint: enable-msg=too-many-arguments
+
```

Comment:
I think it would be much more efficient to just add `self.line = self.framebuf.line` to `__init__` — and same for all of the other functions.